pkg/session/session.go

```diff
@@ -139,6 +197,12 @@ func GetOrCreate(ctx context.Context, params *Params) (*Session, error) {
 	hashedUserPassword := h.Sum(nil)
 	sessionKey := fmt.Sprintf("%s#%s#%s#%x", params.server, params.datacenter, params.userinfo.Username(),
 		hashedUserPassword)
+	sessionOperationMetric.With(prometheus.Labels{
+		metricLabelServer:        params.server,
+		metricLabelDC:            params.datacenter,
+		metricLabelUsername:      params.userinfo.Username(),
+		metricLabelOperationType: metricLabelGetOperation,
+	}).Inc()
```

Comment:
```suggestion
	sessionOperationMetric.WithLabelValues(
		params.server,
		params.datacenter,
		params.userinfo.Username(),
		metricLabelGetOperation,
	).Inc()
```

I think that should work too and we won't need to have the consts.

Prior art: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/pkg/metrics/metrics.go#L86

Reply:
Let's add a util func to increment this metrics. This way it's less verbose here and we don't get the params wrong

Reply:
(if we follow the pattern from elsewhere we end up with that util func basically, e.g. https://github.com/kubernetes-sigs/cluster-api/blob/60317e0fa845109c2ee3c212142e9c210411744b/internal/runtime/metrics/metrics.go#L96)